docker/ansible-in-docker/build.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -vx
+
+function buildAnsible {
+    dockerTag=oso-ansible:${ansibleVersion}
+
+    bashScript=./oso-ansible-playbook-${ansibleVersion}
+
+    docker build --tag ${dockerTag} - << EndOfDockerfile
+FROM fedora:25
+
+RUN yum -y update \
+ && yum -y install ${ansibleRPM} \
+ && yum clean all
+EndOfDockerfile
+
+    cat > ${bashScript} << EndOfBashScript
+#!/bin/bash
+#set -vx
+
+sudo docker run --rm -it --privileged \\
+  -v /etc/ansible:/etc/ansible:ro \\
+  -v /usr/share/ansible/inventory/multi_inventory.py:/usr/share/ansible/inventory/multi_inventory.py:ro \\
+  -v /:/host:ro \\
+  -v /dev/shm/.ansible:/dev/shm/.ansible:ro \\
+  -v \$SSH_AUTH_SOCK:/tmp/sshauth.sock \\
+  -e SSH_AUTH_SOCK=/tmp/sshauth.sock \\
+  -w /host/\`pwd\` \\
+  ${dockerTag} \\
```

Comment:
This should probably be a registry.ops.openshift.com URL so that both tower machines are using the exact same images.